deploy/kubernetes/setup-sumologic.yaml.tmpl

```diff
@@ -7,7 +7,7 @@ kind: ConfigMap
 metadata:
   name:  collection-sumologic-setup
   annotations:
-    "helm.sh/hook": pre-install
+    "helm.sh/hook": pre-install,pre-upgrade
```

Comment:
I think these generated changes were missed in https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/281 which may have been built using an older master with older `build.sh` script